microsoft-edge/progressive-web-apps-chromium/how-to/microsoft-store.md

```diff
@@ -17,41 +17,33 @@ Publishing your Progressive Web App (PWA) to the [Microsoft Store](/windows/uwp/
 | **Trustworthiness** | Windows customers know they can trust their Microsoft Store purchases and downloads, because they adhere to the rigorous Microsoft [quality and safety standards](/legal/windows/agreements/store-policies). |
 | **Easy install** | The Microsoft Store provides a consistent and user-friendly install experience across [all Windows 10 or later apps](https://www.microsoft.com/store/apps/windows). |
-| **App analytics** | The [Windows Partner Center dashboard](/windows/uwp/publish/index) provides you with [detailed analytics](/windows/uwp/publish/analytics) about your app health, usage, and more. |
+| **App analytics** | The [Microsoft Partner Center dashboard](/windows/uwp/publish/index) provides [detailed analytics](/windows/uwp/publish/analytics) about your app's health, usage, and more. |
 
 To publish your PWA to the Microsoft Store, no code changes are required.  Instead, you create an app reservation, package your PWA, and submit your package to the Store.  The following sections explain the steps.
 
 
 <!-- ====================================================================== -->
 ## Create an app reservation
 
-[Windows Partner Center](https://partner.microsoft.com/dashboard/windows/overview) is the hub for you to submit your app to the Microsoft Store.
+To submit your app to the Microsoft Store, you use [Microsoft Partner Center](https://partner.microsoft.com/dashboard/windows/overview).
 
 To create an app reservation:
 
-1.  To display your enrolled programs:
-    1.  Sign into Windows Partner Center with your Microsoft account and go to the [Partner Center Dashboard](https://partner.microsoft.com/dashboard/home).
-    1.  Navigate to **Windows & Xbox**.
-        *   If **Windows & Xbox** is displayed, your app is already enrolled.
-        *   If **Windows & Xbox** isn't displayed, select **Add program**.
+1.  Before you can create an app reservation, you must have a **personal** Microsoft account (not a work Microsoft account or a school Microsoft account) that is enrolled in the Windows Developer Program.
+    1.  If you don't already have one, create a personal Microsoft account at [account.microsoft.com](https://account.microsoft.com/).
+    1.  Enroll in the Windows Developer program, by following the steps in [Open a developer account in Partner Center](/windows/apps/publish/partner-center/open-a-developer-account).
 
-    ![Add a program in the Windows Partner Center dashboard](../media/windows-partner-center-add-program.msft.png)
+1.  Once your account is enrolled in the app developer program, follow these steps to create an app reservation:
+    1.  Open up the [Partner Center Dashboard](https://partner.microsoft.com/dashboard/home). Sign in with the account you enrolled.
```

Comment:
```suggestion
    1.  Go to the [Partner Center Dashboard](https://partner.microsoft.com/dashboard/home). Sign in with the account you enrolled.
```
https://learn.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui - the "Go to" row